app/uk/gov/hmrc/gform/eval/smartstring/SmartStringEvaluatorFactory.scala

```diff
@@ -57,6 +57,13 @@ class RealSmartStringEvaluatorFactory() extends SmartStringEvaluatorFactory {
 
       }
 
+      override def evalEnglish(s: SmartString, markDown: Boolean): String = {
+        val englishImplicitMessages: Messages = englishMessages
+        val englishImplicitLangADT: LangADT = LangADT.En
+
+        self(formModelVisibilityOptics)(englishImplicitMessages, englishImplicitLangADT)(s, markDown)
```

Comment:
Would it be possible to implement `evalEnglish` without need for new `SmartStringEvaluator` instance.

Something like

```
class RealSmartStringEvaluatorFactory(englishMessages: Messages) extends SmartStringEvaluatorFactory {

  def apply(
    formModelVisibilityOptics: FormModelVisibilityOptics[DataOrigin.Mongo]
  )(implicit
    requestMessages: Messages,
    l: LangADT
  ): SmartStringEvaluator =
    new SmartStringEvaluator {

      override def apply(s: SmartString, markDown: Boolean): String =
        withMessages(s, markDown, requestMessages, l)

      override def evalEnglish(s: SmartString, markDown: Boolean): String =
        withMessages(s, markDown, englishMessages, LangADT.En)

      def withMessages(s: SmartString, markDown: Boolean, messages: Messages, lang: LangADT): String = {
        import scala.jdk.CollectionConverters._
        new MessageFormat(s.rawValue(lang))
          .format(
            s.interpolations
              .map { interpolation =>
                formatExpr(interpolation, markDown, messages)
              }
              .asJava
              .toArray
          )
      }
...
````

Reply:
We need to hide implicit message, and lang if we don't use new SmartStringEvaluatorFactory instance. I refactored by putting all method inside `withMessages`. It is a bit ugly now but the other solution when we pass messages and language to all methods explicitly is not better I think. WDYT

Reply:
Hm, I don't like "putting all method inside `withMessages`" nor is my "pass messages and language to all methods explicitly" solution any better, they both includes subtle bugs due to implicits still being in scope. Will push a better solution proposal soon.

Reply:
Here is better suggestion: https://github.com/hmrc/gform-frontend/blob/hide-implicit/app/uk/gov/hmrc/gform/eval/smartstring/SmartStringEvaluatorFactory.scala

Reply:
yes, makes perfect sense. I updated PR with your changes. Thanks.
I still need to wire englishLanguages from ApplicationLoader. Will do that shortly.

Reply:
done englishMessages rewiring form ApplicationLoader